backend/molgenis-emx2-beacon-v2/src/test/java/org/molgenis/emx2/beaconv2/Beaconv2_ModelEndpointsTest.java

```diff
@@ -48,7 +48,7 @@ void testFilteringTerms() throws Exception {
     assertTrue(json.contains("\"type\" : \"ontology\","));
     assertTrue(json.contains("\"id\" : \"NCIT:C124261\","));
     assertTrue(json.contains("\"label\" : \"Whole Transcriptome Sequencing\","));
-    assertTrue(json.contains("\"scope\" : \"sequencingRuns\""));
```

Comment:
don't you prefer the more specific 'sequencingRuns' ?

Reply:
Yes, which is also mentioned in the model itself. But everything else is has Beacon v2 compatible nomenclature now, so renaming this back to "Runs" saves us a headache until we have a good way to perform table/attribute mapping from the base model to Beacon v2.